emergency-funds.md

```diff
@@ -0,0 +1,86 @@
+### Emergency Funds
+
+Before we venture out to any investment such as equity, real estate, gold etc., we need to build a solid emergency fund. Regardless of what job we work on, what our monthly salary is, what is our net worth and so on. 
+
+### What is the need for it
+
+The need for an emergency fund stems from the fact that there is a real emergency happening
+
+- If you are a salaried person then what if you get fired from your job and how will you manage your finances until you get a new one?
+- If you own a business, what if the business is in such a loss and it will not be able to provide for your monthly expenses?
+
+and so on. You should get the idea by now.
+
+### How much should we save
+
+Now we have sufficient evidence that we need an emergency fund, the next question that comes is how much do we need? 
+
+Monthly expense is just one part, there are other emergencies as well but of course we need not get to the point where only the paranoid survive. There is no straight answer to this question since personal finance is personal, but the general rule is to have atleast 
+
+- Cash in had for immediate emergencies (I have arrived at about 50K for this)
```

Comment:
don't use things like 'I'.
Explain the logic also..

Reply:
Done.